core/src/main/java/nl/nn/adapterframework/extensions/fxf/FxfWrapperPipe.java

```diff
@@ -117,7 +119,11 @@ public void configure() throws ConfigurationException {
 			fxfDir = AppConstants.getInstance(getConfigurationClassLoader()).getResolvedProperty("fxf.dir");
 			if (fxfDir == null) {
 				throw new ConfigurationException("property fxf.dir has not been initialised");
-			} else if (!new File(fxfDir).isDirectory()) {
+			}
+			if(isCreateFolder() && !new File(fxfDir).exists() && !new File(fxfDir).mkdirs()) {
+				throw new ConfigurationException(" Cannot create fxf.dir in the path '" + fxfDir + "'");
```

Comment:
Could you please remove the space in front of could (lower case), put arguments between `[ ... ]` instead of `' ...  '`. And can you also change this in the ConfigurationException underneath?